nsq_metrics_to_kairos/nsq_metrics_to_kairos.go

```diff
@@ -68,23 +71,25 @@ func main() {
 	if _, err := os.Stat(*confFile); err == nil {
 		conf, err := globalconf.NewWithOptions(&globalconf.Options{Filename: *confFile})
 		if err != nil {
-			log.Fatal(err)
+			stdlog.Fatal(err)
```

Comment:
Grafana's log library has a Fatal() method.  us it.

`log.Fatal(4, err.Error())`